templates/mod/ssl.conf.erb

```diff
@@ -1,8 +1,16 @@
 <IfModule mod_ssl.c>
   SSLRandomSeed startup builtin
-  SSLRandomSeed startup file:/dev/urandom 512
   SSLRandomSeed connect builtin
+<% if @ssl_random_seeds and ! @ssl_random_seeds.empty? -%>
```

Comment:
this first line is pretty much unnecessary, because
```ruby
Array(undef).each do ..
```
skips the whole loop (and is easier to read).